docs/operating-scylla/security/_common/upgrade-warning-runtime-authentication.rst

```diff
@@ -0,0 +1,20 @@
+
+After Upgrading from 5.4
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The procedure described above applies to clusters where consistent topology updates 
+are enabled. The feature is automatically enabled in new clusters.
```

Comment:
If this applies both to enabling or disabling, how about making it plural, i.e. "The procedures"?